source/precalculus/exercises/outcomes/FN/FN2/template.xml

```diff
@@ -38,17 +40,20 @@
                 <p><m>{{gname}}({{xvalue}})={{gx}}</m></p>
             </outtro>
         </knowl>
+        <!-- {{/findy}}-->
+        <!-- {{#findx}}-->
         <knowl>
             <content>
                 <p>For which <m>x</m>-values in the interval <m>{{interval}}</m> does <m>{{gname}}(x)={{result}}</m>?
```

Comment:
```suggestion
                <p>For which <m>x</m>-value(s) in the interval <m>{{interval}}</m> does <m>{{gname}}(x)={{result}}</m>?
```

Reply:
One little tweak. But looks good to me! Thanks @siwelwerd!

I don't have the review option (perhaps because I did the original PR?), so alerting @StevenClontz to review and approve.